apparmor.d/profiles-m-r/mount

```diff
@@ -9,7 +9,7 @@ abi <abi/3.0>,
 include <tunables/global>
 
 @{exec_path} = /{usr/,}{s,}bin/mount
-profile mount @{exec_path} flags=(complain) {
```

Comment:
Already handled in `main.flags`?

Reply:
They should. But when removing complain flags here please double check that they are already in `main.flags`.

Reply:
Also, you should add all your new profile in `main.flags`.

Reply:
Double checked. Added.